test/integration/cmn/network.go

```diff
@@ -37,27 +43,47 @@ func New(network *nwo.Network, channels []string) *NetworkFoundation {
 		},
 		Channels: channels,
 		Robot:    &Robot{Ports: nwo.Ports{}},
-		mutex:    &sync.Mutex{},
+		ChannelTransfer: &ChannelTransfer{
+			HostAddress: "0.0.0.0",
```

Comment:
Look at the code, somewhere 0.0.0.0.0 is used, somewhere 127.0.0.1 aka loaclhost is used.
Please do the same.